src/graphql.js

```diff
@@ -6,15 +6,15 @@ const {
   ordersTypeDefs,
   paymentsTypeDefs,
   authTypeDefs,
-  leaderboardTypeDefs,
+  //leaderboardTypeDefs,
```

Comment:
I think we can delete this (and the one below too)

Reply:
Looks like it's not in the code... not sure why it's showing up as commented out here